### PR TITLE
Add jax.numpy.bitwise_count

### PR DESCRIPTION
**Pull request by ipranjalks**
Implements `np.bitwiseCount()`, using the SWRA population count technique. The implementation follows JAX in counting the bits of the absolute value.

#### Implementation note

The first SWAR step is written as the masked-add form

    v = (v & 0x55555555) + ((v >> 1) & 0x55555555)

rather than the more common

    v = v - ((v >> 1) & 0x55555555)

because uint32 subtraction currently returns the wrong result on arrays with non-uniform values (#216).

### CODE REVIEW

**Comment by ekzhang** [comment]
Oh interesting do you need this feature? I think this impl is okay, but it's probably better to just implement it in the backend as a Primitive + AluOp, since Wasm has popcnt and WebGPU has countOneBits()

**Comment by ipranjalks** [comment]
Thanks for submitting the fix!
